03/app.js

```diff
@@ -18,12 +18,10 @@ function getSum(a, b, c) {
 	if (b > a && c > a) {
 		return b + c
 	}
-	if (a > c && b > c) {
+	else {
```

Comment:
Tutaj zdecydowanie bez `else` - bez tej opcji jest czytelniej bo to rozwiązanie nie tylko dla tego if-a, ale też związane z pierwszym. Oczywiście to działa - ale może wprowadzić w błąd :)